zc_common/fields.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import unicode_literals
+
+from django.db import models
+from django.core.validators import RegexValidator
+
+
+class PhoneNumberField(models.CharField):
+    description = "A field that checks the value be a valid phone number"
+
+    def __init__(self, *args, **kwargs):
+        phone_regex = RegexValidator(
+            regex=r'^(\+\d{1}\s?)?\(?\d{3}\)?[\s-]?\d{3}[\s-]?\d{4}$', message="Enter a valid Phone Number.")
```

Comment:
Rather than having `phone_regex` re-created each time a `PhoneNumberField` is initialized, it should be a property of the class so it can be be accessed without instantiating the class.